source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -334,6 +363,15 @@ Node Details
   * - DynamicRoute (beta - read only)
     - May have exactly 1 child node. We will evaluate the custom data field value specified on this node's dynamic_route_destination. With non-SIP integration, if the extracted value is a valid phone number and the destination phone number is in an allowed region given your settings, we will play the prompt and transfer the call, otherwise the child node will be executed without the prompt. When SIP integrated, we also allow transferring to any string (such as an extension), in which case the destination should be routable by your SIP infrastructure.
 
+  * - AnyKeyPress
+    - May have exactly 2 child nodes. If any keypress is made, the first child node is executed. If no keypress is made, then the second child node is executed.
+
+  * - NumberQuestion
+    - May have exactly 1 child node. Requires a question type to be selected (e.g. Phone Number, Date). The prompt will play before the caller answers the question. The answer may be saved in a marketing data field.
```

Comment:
This also needs to have a sentence like `Menu` about supporting failover nodes

```
At the end of the child list, it can also optionally have failover child nodes designated by a node with a keypress_failover_type parameter. See example below.
```